addrmgr/network.go

```diff
@@ -118,18 +118,19 @@ func isOnionCatTor(netIP net.IP) bool {
 	return onionCatNet.Contains(netIP)
 }
 
-// NetworkAddress type is used to classify a network address.
-type NetworkAddress int
+// NetAddressType is used to indicate which network a network address belongs
+// to.
+type NetAddressType uint8
 
 const (
-	LocalAddress NetworkAddress = iota
+	LocalAddress NetAddressType = iota
 	IPv4Address
 	IPv6Address
-	OnionAddress
+	TORv2Address
 )
 
 // getNetwork returns the network address type of the provided network address.
-func getNetwork(netIP net.IP) NetworkAddress {
+func getNetwork(netIP net.IP) NetAddressType {
```

Comment:
Updated.  There are changes in a later pull request to calculate and store this value on the addrmgr.NetAddress type as on construction of the addrmgr.NetAddress, but I'm holding off on back porting that for now.  Let me know if it would be desirable for that to be pulled into this commit or PR.  Otherwise, I'll just stick with renaming the function for now as suggested.